src/applications/disability-benefits/2346/prefill-transformer.js

```diff
@@ -0,0 +1,31 @@
+import _ from '../../../platform/utilities/data';
+import { disabilityActionTypes, SERVICE_CONNECTION_TYPES } from './constants';
+
+export const filterServiceConnected = (disabilities = []) =>
+  disabilities.filter(
+    d => d.decisionCode === SERVICE_CONNECTION_TYPES.serviceConnected,
+  );
+
+export const addNoneDisabilityActionType = (disabilities = []) =>
+  disabilities.map(d =>
+    _.set('disabilityActionType', disabilityActionTypes.NONE, d),
+  );
+
+export const setClaimTypeNewOnly = formData =>
+  _.set(
+    ['view:claimType'],
+    {
+      'view:claimingNew': true,
+      'view:claimingIncrease': false,
+    },
+    formData,
+  );
+
+export function prefillTransformer(pages, formData, metadata) {
+  const newData = formData;
```

Comment:
I don't think this is needed 🤔

Reply:
@kelsonic  This whole file is not needed we are getting rid of it.

Reply:
This has been deleted.